cluster/charts/rook-ceph/templates/deployment.yaml

```diff
@@ -134,6 +134,22 @@ spec:
           value: {{ .Values.csi.attacher.image | quote }}
 {{- end }}
 {{- end }}
+{{- if .Values.csi.provTolerations }}
```

Comment:
Please spell out `provisioner` instead of abbreviating it in all these helm and env vars to help clarify to the user.